common/document_parser/lib/responsibility_parse/responsibility_parser.py

```diff
@@ -12,22 +12,34 @@
 # gamechanger-data imports
 from common.document_parser.cli import get_default_logger
 from common.document_parser.lib.document import FieldNames
-
+punctuation_less_period_parentheses = set(string.punctuation).difference({".","(",")"})
+
+numbering_regex = "^([a-z]{1,2}\.|" \
+                          "\([a-z]{1,2}\)|" \
+                          "\(\d{1,2}\)|" \
+                          "\d{1,2}\.*|" \
+                          "\d{1,2}\.\d{1,2}\.*|" \
+                          "\d{1,2}\.\d{1,2}\.\d{1,2}\.*|" \
+                          "\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.*|" \
+                          "\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.*|" \
+                          "\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.*|" \
+                          "\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.\d{1,2}\.*)$"
```

Comment:
Similar to the comments I made on `is_role_acronym_defined()`:
- If the capture groups aren't used anywhere, I suggest converting them to non-capture groups
- Since this pattern doesn't change, I suggest compiling it once when it's defined here